### ISSUE TITLE
Additions to residual velocity magnitude scatter plot

### ISSUE THREAD

**Issue by brendanjmeade** [brendanjmeade/result_manager#18]
Thoughts @jploveless?

- Scale scatter plot circle marker size by residual area so that the largest residuals are easy to identify?
- Colorbar?

**Comment by jploveless** [comment]
I really like this idea — double the visual cues! Do you think these should have their size changed by the slider/plus-minus buttons? 

I will try to work on colorbars soon. I don't think we ever had one for the residual magnitudes for ResultManager, as the color range was set to span 0–5 mm/yr. For slip rates, the colorbar was adjusted based on the actual range of values of the currently plotted quantities, but I'd really like to create an editable range here (currently it's hard coded to be -100 to 100 mm/yr). That feels a little complicated at this point, so it may be a while before I get to it.

**Comment by brendanjmeade** [comment]
I agree that double visual cues is a good idea.  The other thing that might be useful is a button that generates pop-up that lists the 10 stations with the largest residuals/contribution to the minimization criteria.

**Comment by jploveless** [comment]
I like the idea of the 10 largest contributions being more clearly identified. The pop-up could be a handy way of seeing this, perhaps supported by a red (or other bright color) outline around the large dots.
